pages/k8s/install-manual.md

```diff
@@ -0,0 +1,288 @@
+repeatable than customising a deployment post-install.
+
+This documentation first presents the method for installing from the official release bundles, then explains how the bundle can be customised.
+
+
+## Install CDK from the official bundle
+
+The following sections outline a standard installation of CDK using the stable release Juju bundles. The standard bundle includes all the components of Kubernetes, but you will have to also follow the [additional configuration][cloud-specific] steps at the end for Kubernetes to function properly - additional components are required to enable Kubernetes to interact with the cloud it is deployed on.
+
+###  Install Juju
+
+If Juju has not already been installed on your system, you need to install it first. For Ubuntu 16.04 and later, and other operating systems which support [snaps][snaps], run the command:
+
+```bash
+sudo snap install juju --classic
+```
+
+For other install options, please see the [Juju documentation][juju-docs].
+
+### Create a controller
+
+Juju requires a *controller*  instance to manage models and deployed applications.
+You can make use of a hosted controller ([JAAS][jaas]) or create one in a cloud of
+your choice. For public clouds, you will need to have added a credential first:
+
+```bash
+juju add-credential
+```
+
+...will step through adding your credential for a specific cloud.
+
+
+The Juju documentation has more information on [adding credentials][credentials] and [configuring a controller][controller-config]
+
+<div class="p-notification--caution">
+  <p markdown="1" class="p-notification__response">
+    <span class="p-notification__status">Note:</span>
+To manually install CDK locally using LXD, it is necessary to use a specific container profile before deploying.  See the <a href="/kubernetes/clouds-lxd">notes on LXD</a> before deployment.
+  </p>
+</div>
+
+
+
+### Create a model
+
+The controller automatically creates a model named 'default'. It is useful to have a different model for each deployment of CDK, and for the models to have useful names. You can create a new model with the `add-model` command like this:
+
+```bash
+juju add-model cdk-test
+```
+
+...susbstituting the desired name.
+
+### Deploy the CDK bundle
+
+The Juju Charm Store hosts the CDK bundles as well as individual charms.  To deploy
+the latest, stable bundle, run the command:
+
+```bash
+juju deploy cs:~containers/canonical-kubernetes
+```
+
+It is also possible to deploy a specific version of the bundle by including the
+revision number. For example, to deploy the CDK bundle for the Kubernetes 1.12
+release, you could run:
+
+```bash
+juju deploy cs:~containers/canonical-kubernetes-296
+```
+
+The revision numbers for bundles are generated automatically when the bundle is
+updated, including for testing and beta versions, so it isn't always the case
+that a higher revision number is 'better'. The revision numbers for the release
+versions of the CDK bundle are shown in the table below:
```

Comment:
The page you got these from was out of date. I've updated https://github.com/juju-solutions/bundle-canonical-kubernetes/wiki to have the accurate list.

Reply:
okay, I updated the table. Just a few more things to fix